smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/integration/middlewares/LoggingMiddleware.kt

```diff
@@ -29,14 +29,10 @@ class LoggingMiddleware : MiddlewareRenderable {
         op: OperationShape,
         operationStackName: String
     ) {
-        writer.write("$operationStackName.${middlewareStep.stringValue()}.intercept(position: ${position.stringValue()}, middleware: \$N(${middlewareParamsString(model, symbolProvider, op)}))", ClientRuntimeTypes.Middleware.LoggerMiddleware)
+        writer.write("$operationStackName.${middlewareStep.stringValue()}.intercept(position: ${position.stringValue()}, middleware: \$N(${middlewareParamsString()}))", ClientRuntimeTypes.Middleware.LoggerMiddleware)
     }
 
-    override fun middlewareParamsString(
-        model: Model,
-        symbolProvider: SymbolProvider,
-        op: OperationShape
-    ): String {
+    private fun middlewareParamsString(): String {
```

Comment:
if we are removing this, might as well jsut remove the whole need for another function to just return a string. can you just inline this into line 32?

Reply:
spoke offline. keeping this as-is